recipes/boost/all/conanfile.py

```diff
@@ -1116,7 +1109,7 @@ def add_defines(library):
             if self.options.multithreading:
                 cxx_flags.append("-DBOOST_SP_USE_SPINLOCK")
 
-            if "embed_bitcode" in self.options and self.options.embed_bitcode:
+            if self.conf.get("tools.apple:enable_bitcode", check_type=bool):
```

Comment:
Remember that, whenever this recipe will be migrated, we should remove these lines because, with that configuration enabled, `CMakeToolchain` will automatically add that bitcode flag.

Reply:
boost is not built with CMake (yet?), but b2.

Reply:
I'd put money on Boost _never_ switching from `b2` to `CMake`.